internal/model/apiconfig.go

```diff
@@ -30,35 +30,26 @@ const (
 // application and their access to and requirements for using the API.
 // DB times of 0 are interpreted to be "unbounded" in that direction.
 type APIConfig struct {
-	AppPackageName    string          `db:"app_package_name"`
-	Platform          string          `db:"platform"`
-	ApkDigestSHA256   []string        `db:"apk_digest"`
-	CTSProfileMatch   bool            `db:"cts_profile_match"`
-	BasicIntegrity    bool            `db:"basic_integrity"`
-	AllowedPastTime   time.Duration   `db:"allowed_past_seconds"`
-	AllowedFutureTime time.Duration   `db:"allowed_future_seconds"`
-	AllowedRegions    map[string]bool `db:"allowed_regions"`
-	AllowAllRegions   bool            `db:"all_regions"`
-
-	// BypassSafetyNet is an internal field for testing that bypasses Android
-	// SafetyNet verification. It is not read from a database and is used for
-	// testing only.
-	BypassSafetyNet bool
-
-	// BypassDeviceCheck is an internal field for testing that bypasses iOS
-	// DeviceCheck verification. It is not read from a database and is used for
-	// testing only.
-	BypassDeviceCheck bool
+	AppPackageName    string
+	Platform          string
+	ApkDigestSHA256   []string
```

Comment:
should we group the android fields like the iOS fields are?

Reply:
Done